### DISCUSSION TITLE
Welcome to python Discussions!

### DISCUSSION THREAD

**Discussion by yasenstar** [yasenstar/python#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by goldenflo** [comment]
I'm enrolled in a **[Python online training]** program in the USA with skilled and experienced trainers. They teach in a friendly and supportive manner, with 24/7 accessibility. The course lasts for three months, helping me enhance my skills effectively[](.https://vytcdc.us/python-online-training``)

**Reply by yasenstar** [reply]
Thanks for information, agree, enrolling in the training would be one "force" approach to learn, after that, keep practicing the learnt skills in daily manner will be more important, good luck!
